WooCommerce/Classes/ViewRelated/Products/SKU Scanner/ProductSKUInputScannerViewController.swift

```diff
@@ -59,8 +59,9 @@ private extension ProductSKUInputScannerViewController {
 
 private extension ProductSKUInputScannerViewController {
     enum Localization {
-        static let title = NSLocalizedString("Scan barcode to update SKU", comment: "Navigation bar title for scanning a barcode to use as a product's SKU.")
-        static let instructionText = NSLocalizedString("Scan product barcode",
+        static let title = NSLocalizedString("Scan barcode or QRCode to update SKU",
+                                             comment: "Navigation bar title for scanning a barcode or QRCode to use as a product's SKU.")
+        static let instructionText = NSLocalizedString("Scan product barcode or QRCode",
```

Comment:
Super nit: Now that we are updating the strings, consider using [reverse DNS keys](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/localization.md#use-unique-reverse-dns-naming-style-keys) for the localized strings.

Reply:
Done a2478aa8d5354b41a1ce2199941eb6c4bffa53dc